### ISSUE TITLE
gui.mouseEvents: true disables mouse, not gui.ignoreMouseEvents: true

### ISSUE THREAD

**Issue by valankar** [jesseduffield/lazydocker#480]
**Describe the bug**
`gui.mouseEvents: true` is what is used to disable the mouse. The documentation states it should be `gui.ignoreMouseEvents` but that's not the case. Also `gui.mouseEvents: true` sounds like it should *enable* the mouse, but it actually disables it.

**Desktop (please complete the following information):**
 - OS: Linux
 - Lazydocker Version 0.21.1

**Comment by BradLewis** [comment]
I agree this is confusing. I can see 4 years ago the flag was [changed](https://github.com/jesseduffield/lazydocker/commit/27eb5a6474714c99a9c21bf8acbc3248476282d9#diff-5a58a868e044c5811587ec95ac0793ef29023fe00d03f0ab88fa3e6ef0534cc8 ) internally from `MouseEvents` to `IgnoreMouseEvents` and the logic inverted. However the yaml option wasn't changed from `mouseEvents`. 

I'm not sure how you'd change this now without maintaining backwards compatibility? Could possibly create a new option called `ignoreMouseEvents` and deprecate `mouseEvents`.  Could also just improve the docs to make it more clear.
Would be an easy fix either way, just depends on how they'd want to tackle it.

**Comment by nikhilweee** [comment]
I would very much support this. The current option is very confusing.
